middleware/index.ts

```diff
@@ -168,6 +168,22 @@ function createContext(id: string, logger: Logger, reqContext: any, accessToken?
     return context;
 }
 
+/**
+ * TODO: replace this with salesforce-sdk api call to get secret
+ * @param name
+ * @param key
+ * @param logger
+ */
+function getSecret(name, key, logger: Logger) : string {
+    try {
+        let data = fs.readFileSync(`/platform/services/${name}/secret/${key}`);
+        return data.toString();
+    } catch(err) {
+        logger.info(`Failed to read secret name "${name}" key "${key}": ${err.stack}`);
```

Comment:
probably `logger.debug` depending on how often we're calling it.

Reply:
Also, suggest not logging `stack`.

Reply:
@dhagberg-sf , logger.debug will never log anything in a deployed environment, as there is no way to set that DEBUG=1 runtime env in deployed evergreen...hence info